datalad/support/exceptions.py

```diff
@@ -0,0 +1,57 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+""" datalad exceptions
+"""
+
+
+class CommandError(RuntimeError):
```

Comment:
may be (just may be) worth making more verbose?  CommandRunError, or ExternalCommandError?